lua/lspconfig/pasls.lua

```diff
@@ -0,0 +1,50 @@
+local configs = require 'lspconfig/configs'
+local util = require 'lspconfig/util'
+
+configs.pasls = {
+  default_config = {
+    cmd = { 'pasls' },
+    cmd_env = {
+      FPCDIR = '/usr/lib/fpc/src',
+      PP = '/usr/lib/fpc/3.2.2/ppcx64',
```

Comment:
These paths are too specific. Does the language server really not work without these?

Reply:
The only Path required for the server to work is `FPCDIR` as i specified in the documentation now.

Reply:
Wouldn't it be better if these were inherited from the parent environment rather than set by us? That feels more flexible to me, and kind of the point of environment variables. So you'd just stick `export FPCDIR='...'` in your `.bashrc` or similar.

Reply:
I was going to make it that way at first but now that i think about it it's better this way.